### ISSUE TITLE
Question: Static

### ISSUE THREAD

**Issue by gesker** [melloware/quarkus-faces#286]
Good Morning @melloware 

Was just wondering if you have tried preparing this project with **_--static and --libc=musl_** and/or  **_-H:+StaticExecutableWithDynamicLibC_** as in this [guide](https://www.graalvm.org/latest/reference-manual/native-image/guides/build-static-executables/)?

Just curious.

**Comment by melloware** [comment]
Nope I just defer to Mandrel which is the Quarkus optimized version of GraalVM from RedHat? I assume they have all the tuning and tweaks built in already but I am no expert on the GraalVM stuff.  That would be @Karm who is the expert on this stuff. 😄

**Comment by gesker** [comment]
I saw that guide and played with it a bit on a smaller unrelated project - just to try it. But, at that time I didn't see any documentation over on the Quarkus side for those switches and so shied away from it. 

Generally, I try to stick with the container build, too - which is nice BTW - as my env variables move a lot as I project hop through out the day/week.  

Then, I was back to this project for reference (completely different item) and remembered that you had some speeds and sizes recorded in the read me. I found myself wondering if those speeds were because of these switches.

Anyway, thanks again for THIS reference project!
